### ISSUE TITLE
QuickFIX doesn't support for python3+?

### ISSUE THREAD

**Issue by yutiansut** [quickfix/quickfix#153]
```

      print "Testing for std::tr1::shared_ptr..."
                                                  ^
    SyntaxError: Missing parentheses in call to 'print'
```

error in setup period, seems like a python2 syntax

**Comment by joncinque** [comment]
I've just put in a pull request to support it, but it may take some time before it's integrated. It's at #158 if you want to follow

**Comment by yutiansut** [comment]
@joncinqe thanks a lot～～

**Comment by mkst** [comment]
@yutiansut Are you happy to close this issue? PR was merged 2.5 years ago :)
